object/host_system.go

```diff
@@ -32,12 +32,31 @@ type HostSystem struct {
 	InventoryPath string
 }
 
+func (h HostSystem) String() string {
```

Comment:
While the name is human readable, it might be ambiguous if there is not enough context (i.e. multiple objects with the same name in different folders). The reference -- which is already available on these objects -- is an unambiguous reference to these objects. It's less human friendly, though still contains both the object type and an identifier. What do you think of using the reference instead? That way it is even possible to only implement `String()` once on the `Common` struct.